### PR TITLE
Fixed Typos in intro.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mdxabu**
Dear Maintainer,

I wanted to inform you that I have made some typos and grammar corrections in the ```CONTRIBUTING.md``` and ```intro.md```  file. I have opened a PR for the changes to be reviewed and merged.

Best regards,
F Mohamed Abdullah

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ with your real name (first and last name)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`. If you forgot this you can
 use `git commit -s --amend` to add this in retrospective for the last commit.
-If you need to sign-off multiple commits within a branch, you need to do an interactive
+If you need to sign off multiple commits within a branch, you need to do an interactive
```
Comment (rohanKanojia) [inline CONTRIBUTING.md:71]: Please keep `sign-off` . It implies user to use `git commit --sign-off`

Comment (mdxabu) [reply CONTRIBUTING.md:71]: ok

Comment (mdxabu) [reply CONTRIBUTING.md]: I changed that @rohanKanojia

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,22 +34,22 @@ license document, but changing it is not allowed.
 
 Developer's Certificate of Origin 1.1
 
-By making a contribution to this project, I certify that:
+By contributing to this project, I certify that:
```
Comment (rohanKanojia) [inline CONTRIBUTING.md:37]: I think this is directly taken from https://developercertificate.org/

I would not modify text in this code block. It should exactly match it's source.

**Comment by rohanKanojia** [comment]
@mdxabu Could you please squash your commits into one?

**Comment by mdxabu** [comment]
> @mdxabu Could you please squash your commits into one?

@rohanKanojia I Squashed those commits into one commit.

**Comment by rohanKanojia** [comment]
@mdxabu : Thanks, could you please address https://github.com/fabric8io/docker-maven-plugin/pull/1735#discussion_r1427552998 as well?

**Comment by mdxabu** [comment]
> @mdxabu : Thanks, could you please address https://github.com/fabric8io/docker-maven-plugin/pull/1735#discussion_r1427552998 as well?

Ok

File: CONTRIBUTING.md (hunk 2 above)
Comment (mdxabu) [reply CONTRIBUTING.md]: I'm working on it.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,19 +37,19 @@ Developer's Certificate of Origin 1.1
...
     license and I have the right under that license to submit that
     work with modifications, whether created in whole or in part
-    by me, under the same open source license (unless I am
+    by me, under the same open-source license (unless I am
     permitted to submit under a different license), as indicated
     in the file; or
 
 (c) The contribution was provided directly to me by some other
-    person who certified (a), (b) or (c) and I have not modified
+    the person who certified (a), (b) or (c) and I have not modified
```
Comment (rhuss) [inline CONTRIBUTING.md:52]: This needs to be corrected. And please, as @rohanKanojia mentioned, don't touch the Developer's Certificate of Origin part as we do not own this; we just replicate it here. If you want to update this, please get in touch with the Linux Foundation, when they change it, we will take it over here. Consider it like a law text, you can't change it even with errors.

**Comment by rhuss** [comment]
@mdxabu thanks for the PR! Looks good to me, but please leave the part untouched that we copy over from somewhere. See my inline comment.

File: CONTRIBUTING.md (hunk 3 above)
Comment (mdxabu) [reply CONTRIBUTING.md:52]: > This needs to be corrected. And please, as @rohanKanojia mentioned, don't touch the Developer's Certificate of Origin part as we do not own this; we just replicate it here. If you want to update this, please get in touch with the Linux Foundation, when they change it, we will take it over here. Consider it like a law text, you can't change it even with errors.

I deleted that commit that I changed that Developer's Certificate. Now I just made changes only in ```into.md``` and ```CONTRIBUTING.md```

**Comment by mdxabu** [comment]
> @mdxabu thanks for the PR! Looks good to me, but please leave the part untouched that we copy over from somewhere. See my inline comment.

I Changed that please go through it if any queries please ping me here.

**Comment by mdxabu** [comment]
> > @mdxabu thanks for the PR! Looks good to me, but please leave the part untouched that we copy over from somewhere. See my inline comment.
> 
> I Changed that please go through it if any queries please ping me here.

**Comment by rhuss** [comment]
Thanks, looks good now! But it looks also like you closed the PR or deleted the original repository. 

wrt/ Discussions: Sorry, we don't have yet but that is probably a good idea to enable GitHub discussions. Wdyt, @rohanKanojia ?

**Pull request closed without merging**